modules/mono/glue/GodotSharp/GodotPlugins/Main.cs

```diff
@@ -238,11 +238,11 @@ private static bool UnloadPlugin(ref PluginLoadContextWrapper? pluginLoadContext
 
                 if (!pluginLoadContext.IsCollectible)
                 {
-                    Console.Error.WriteLine("Cannot unload a non-collectible assembly load context.");
+                    WriteLineError("Cannot unload a non-collectible assembly load context.");
                     return false;
                 }
 
-                Console.WriteLine("Unloading assembly load context...");
+                WriteLine("Unloading assembly load context...");
```

Comment:
Using `Console.WriteLine` here may have been on purpose. It looks like a verbose message to me, so printing only to the console should be fine. Alternatively, it probably should only be printed when `OS.IsStdOutVerbose()` is true.